.github/workflows/nodejs.yml

```diff
@@ -0,0 +1,46 @@
+name: Node.js CI
```

Comment:
This change is out of context.

Reply:
File removed
https://github.com/schul-cloud/node-notification-service/pull/205/commits/ed12ab488e29e057410f4a904dccf68a71e107c9